baseline-refaster-testing/build.gradle

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'nebula.maven-publish'
```

Comment:
ah also don't apply this, publish-jar does the right thing here

Reply:
because of https://github.com/palantir/gradle-baseline/blob/9efd8cca8523d17ab6ee19f78898755342ad6f56/gradle/publish-jar.gradle#L3-L4